app/src/main/java/org/lineageos/tv/launcher/notification/NotificationAdapter.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * SPDX-FileCopyrightText: 2024 The LineageOS Project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.lineageos.tv.launcher.notification
+
+import android.annotation.SuppressLint
+import android.app.Notification
+import android.content.Context
+import android.service.notification.NotificationListenerService.RankingMap
+import android.service.notification.StatusBarNotification
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.LinearLayout
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import org.lineageos.tv.launcher.R
+
+class NotificationAdapter(
+    private val context: Context,
+    private val onItemClickListener: OnItemClickListener
+) :
+    RecyclerView.Adapter<RecyclerView.ViewHolder>() {
```

Comment:
`: RecyclerView.Adapter<RecyclerView.ViewHolder>() {`

Reply:
Try to use ListAdapter btw, handles updates sanely, especially with classes with ids